BlueprintUICommonControls.podspec

```diff
@@ -14,4 +14,9 @@ Pod::Spec.new do |s|
   s.source_files = 'BlueprintUICommonControls/Sources/**/*.swift'
 
   s.dependency 'BlueprintUI'
+
+  s.test_spec 'Tests' do |test_spec|
```

Comment:
Gets these tests showing up in Cocoapods.

Reply:
Yeah – but now you can't run them locally with Cocoapods. Also seems wise to run the tests in the Cocoapods environment in case there are resource, compilation issues, etc.

Reply:
The reason they were removed from the spec is that they require iOS 13, but `pod lib lint` and friends will execute them using a simulator for the minimum deployment target. So they will always cause linting to fail.

If you want to run them from Xcode, I recommend opening the project with the Package.swift file. It's nice!

Reply:
I've been doing that, but I really want them running on CI as well, since we do only consume this stuff via Cocoapods... Hmmm...